src/components/RegisterForm/RegisterForm.jsx

```diff
@@ -11,16 +11,18 @@ import OutlinedInput from '@mui/material/OutlinedInput';
 import Stack from '@mui/material/Stack';
 import TextField from '@mui/material/TextField';
 import Box from '@mui/system/Box';
-import { useState } from 'react';
+import { useContext, useState } from 'react';
 import { Controller, useForm } from 'react-hook-form';
 
 import styles from '/src/components/RegisterForm/RegisterForm.module.css';
+import LoginContext from '/src/contexts/LoginContext';
 import { registerSchema } from '/src/schemas/authSchemas';
 import { SIGN_UP_URL } from '/src/URLs';
 import { signInSignUp } from '/src/utils/auth';
 
 function RegisterForm(props) {
   const { fn, ...otherProps } = props;
+  const login = useContext(LoginContext);
```

Comment:
I think you could destructure `{ loggedIn, login, logout }` right away as well as in the login form.